docs/local-backend-roles/localroles.md

```diff
@@ -0,0 +1,29 @@
+# Setting roles in your local backend
+
+Please make sure that you have setup the backend locally. Watch these videos if you have not done that: 
+- <a href="https://www.youtube.com/watch?v=haqPaPRrhPU">Video 1</a>
+- <a href="https://www.youtube.com/watch?v=2Ja5hH2nH1o&feature=youtu.be">Video 2</a>
+
+## Why we need to step up roles in our local backend?
+- There might be a certain feature that you are building which requires admin or super_user role in order to perform a particular action.
+- Let's take an example: suppose you are working on a feature which allows the super_user to promote a user of Real Dev Squad to member, for that you need to hit `/moveToMembers/:username` endpoint, it uses a authorizeUser middleware to check if the user has a super_user role. For that you need to use your local backend and set up the required roles in firebase to provide super_user role to yourself.
+
+## Steps: 
+
+1. Make sure that your backend server is running. To run the server run `yarn dev` in your project directory. This will run your server on localhost:3000
+
+2. The RDS backend hits `/users/self` endpoint to get the data of the user.
+
+3. So let's try to acces our details by going to http://localhost:3000/users/self. This will give a 401 error, to solve this follow the step below.
+
+4. Copy this URL `https://github.com/login/oauth/authorize?client_id=<your-client-id>` and paste it somewhere, now go to your  `local.js` file and copy the clientId that is present there, and paste it in place of `<your-client-id>` in the URL provided above.

```

Comment:
Suggestion: We can also add a link to GitHub docs explaining how to get GitHub client ID directly from [your application's configuration page](https://github.com/settings/developers).

Reply:
Do we really need to do that? Someone who has set up the backend has that in their local.js and people who have not set up the backend need to set it up first

Reply:
For developers who are just going though the docs before setting up the backend might not have the context of `local.js` file. I think for them GitHub docs might make sense.

Reply:
makes sense. I will do that